# Desired distance from grasp_postures_frame_id to put the object
# So, the grasping point
grasp_desired_distance: 0.20
grasp_min_distance: 0.0
# Times for the grasp, they are sumed step after step
time_pre_grasp_posture: 2.0
time_grasp_posture: 1.0
time_grasp_posture_final: 3.0
# Base frame of MoveIt for your robot
grasp_pose_frame_id: base_footprint
# Frame where the direction of the poses will be computed
grasp_postures_frame_id: arm_tool_link
# Currently unused grasp quality to put in the message
grasp_quality: 0.1
# Gripper joint poses
gripper_joint_names: "gripper_left_finger_joint gripper_right_finger_joint"
# Closed position
gripper_grasp_positions: "0.015 0.015"
# Open position
gripper_pre_grasp_positions: "0.038 0.038"

# Direction in reference of the grasp_postures_frame_id
pre_grasp_direction_x: 1.0
pre_grasp_direction_y: 0.0
pre_grasp_direction_z: 0.0

# Direction in reference of the grasp_postures_frame_id
post_grasp_direction_x: -1.0
post_grasp_direction_y: 0.0
post_grasp_direction_z: 0.0

max_contact_force: 0.0
allowed_touch_objects: ''

# Rotation to apply if you have unaligned
# tool link to grasping frame
fix_tool_frame_to_grasping_frame_roll: -90.0
fix_tool_frame_to_grasping_frame_pitch: 0.0
fix_tool_frame_to_grasping_frame_yaw: 0.0

# Generation of grasps configuration
# in order to generate more or less poses these
# parameters can be tuned
min_degrees_yaw: 0
max_degrees_yaw: 360
min_degrees_pitch: 0
max_degrees_pitch: 360

# The steps are the main thing to tune
step_degrees_pitch: 15
step_degrees_yaw: 15

links_to_allow_contact: ["gripper_left_finger_link", "gripper_right_finger_link", "gripper_link"]
